README.md

```diff
@@ -126,6 +189,76 @@ client.on('connected', async (conn) => {
 client.start();
 ```
 
+# PulseQueue
+
+A PulseQueue declares a queue and listens for messages on that
+queue, invoking a callback for each messages.
+
+The options to the constructor are:
+
+```javascript
+let pq = new pulse.PulseQueue({
+  client,                // Client object for connecting to the server
+  bindings: [{           // exchange/routingKey patterns to bind to
+    exchange,            // Exchange to bind
+    routingKeyPattern,   // Routing key as string
+    routingKeyReference, // Reference used to parse routing keys (optional)
```

Comment:
Unrelated, but maybe we should rename these to `pattern` and `reference` instead of `routingKeyPattern`...

We can just leave `routingKeyPattern` supported, but not documented that way existing code will keep working.